R/write_secuTrial.R

```diff
@@ -1,11 +1,13 @@
 #' Write secuTrial exports to other formats
-#' @description Convert the export prepared in R and export it SPSS, Stata or SAS using the haven package.
+#' @description Convert the export prepared in R and export it to
+#'              SPSS (sav), Stata (dta) or SAS (sas, xpt version 8)
+#'              using the haven package.
 #' @name write_secuTrial
 #' @param object secuTrialdata object
 #' @param path directory where the files should be saved
-# @param format format in which to save the export (one of "dta", "sas", "sav", "xpt")
-#' @param format format in which to save the export (one of "dta", "sas", "sav")
-#' @details Due to variable naming limitations in other packages, date variables are appended with _d (rather than _date), datetime/POSIX variables are appended with _dt (rather than _datetime) and factors with _f (rather than _factor).
```

Comment:
I would actually retain this as it's secuTrialR not haven... or did you remove that functionality? haven might do other stuff though...

Reply:
Ah, I didn't realize we were doing that. I will put it back in.

Reply:
Thanks